sys/arm64/include/_align.h

```diff
@@ -38,7 +38,7 @@
  * for all data types (int, long, ...).   The result is unsigned int
  * and must be cast to any desired pointer type.
  */
-#define	_ALIGNBYTES	(sizeof(long long) - 1)
+#define	_ALIGNBYTES	(sizeof(void *) - 1)
```

Comment:
This is not relevant for a hybrid kernel

Reply:
Dropping; it belongs in a future purecap PR.

Reply:
Are you dropping this or not? I think keeping it here seems fine.

Reply:
I thought I had a comment saying I was going to keep it, oh well. But yes, it's staying because it's a kernel header that facilitates userland parts to build correctly, I just made sure to fix the commit message to accurately state what it actually achieves.